datafusion/functions-aggregate/src/approx_distinct.rs

```diff
@@ -304,3 +223,88 @@ where
 
     default_accumulator_impl!();
 }
+
+impl Debug for ApproxDistinct {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("ApproxDistinct")
+            .field("name", &self.name())
+            .field("signature", &self.signature)
+            .finish()
+    }
+}
+
+impl Default for ApproxDistinct {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+pub struct ApproxDistinct {
+    signature: Signature,
+    alias: Vec<String>,
```

Comment:
ApproxDistinct has no any alias. I think we can just remove this field and use default `AggregateUDFImpl::alias`

Reply:
Sure